scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -75,6 +89,36 @@ label {
   }
 }
 
+// Style new group of type radio
+.form-inline {
+
+  &.form-inline-tabs .form-group {
+    box-shadow: $crm-form-control-shadow;
+    border: 1px solid $crm-darker-grayblue;
+    line-height: 1em;
+    padding: 8px 15px;
+    margin-right: -5px;
```

Comment:
Does that margin come from somewhere (as in, was it inherited)? or is it something that happen because the elements are inline?

any solution that relies either on a non-numeric value property (like `float`) or that it uses a value that comes from a shared variable (say something like `margin-right: -#{$some-common-margin-value}`) is preferrable over a solution with arbitrary numbers